pandas/tests/frame/test_duplicates.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import numpy as np
+from pandas import Series, DataFrame
+
+from pandas.util.testing import assert_series_equal, assert_frame_equal
+import pandas.util.testing as tm
+
+
+class TestDataFrameDuplicated(object):
+
+    def test_duplicated_keep(self):
+        df = DataFrame({'A': [0, 1, 1, 2, 0], 'B': ['a', 'b', 'b', 'c', 'a']})
+
+        # keep = 'first'
+        exp = Series([False, False, True, False, True])
+        assert_series_equal(df.duplicated(keep='first'), exp)
```

Comment:
My preference would be to use a slightly more explicit pattern throughout these tests along the lines of:
```python
expected = ...
result = ...
assert_series_equal(expected, result)
```